CMakeLists.txt

```diff
@@ -0,0 +1,204 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8.13)
+
+# Configuration options
+option(vcdiff_build_tests "Build all of vcdiff tests." ON)
```

Comment:
I must apologize for this as it does not look like we have a public styleguide for cmake files. You can take a look at the gflags CMakeLists.txt file for reference but essentially, you should use lowercase and precede the opening paren with a space for all the CMake functions, e.g.
```
if (vcdiff_build_exec)
  set (GFLAGS_BUILD_gflags_LIB OFF)
```
instead of:
```
IF(vcdiff_build_exec)
  SET(GFLAGS_BUILD_gflags_LIB OFF)
```